src/test/java/com/datastax/oss/protocol/internal/RequestFrameCodecTest.java

```diff
@@ -195,7 +200,10 @@ public int encodedSize(Message message) {
     @Override
     public <B> Message decode(B source, PrimitiveCodec<B> decoder) {
       assertThat(decoder.readString(source)).isEqualTo(MOCK_ENCODED);
-      return Options.INSTANCE;
+      List<String> eventTypes =
```

Comment:
Probably not the most elegant way, since it's also defined in the Parent class too, but it got the job done. I'm open to other suggestions if there's a better way.

Reply:
That will work, I'll just extract the list of events in a constant.